Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN INTERACTIVE=false CI=true MB_EDITION=$MB_EDITION bin/build.sh :version ${VER
 ## jar from the previous stage rather than the local build
 ## we're not yet there to provide an ARM runner till https://github.com/adoptium/adoptium/issues/96 is ready
 
-FROM --platform=linux/amd64 eclipse-temurin:11-jre-alpine as runner
+FROM --platform=linux/amd64 eclipse-temurin:21-jre-alpine as runner
```

Comment:
Just a comment: as soon as we release our first ARM image, we will be able to remove the platform specification from both Dockerfiles so the docker engine can start pulling the correct one based on host OS cc @iethree

Reply:
Fixed by #48910